.github/workflows/Build_and_Test.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Test
```

Comment:
This project is built using maven not sbt. Please check the readme file on the project.
https://github.com/CareCloud/Data-Science-Extensions/tree/master/spark-datasource-rest

This is the command to build/install it locally:
`mvn clean install -pl spark-datasource-rest -DskipTests -Dscalastyle.skip=true`

You will have to use a similar approach to the ones used in the legacy edi projects
https://github.com/CareCloud/edi_core/blob/master/.github/workflows/publish.yml

@AsfaMumtaz24 has been doing most of the development in this project. Please reach out to her if you have any questions.